src/main/java/ToDoCommand.java

```diff
@@ -0,0 +1,39 @@
+public class ToDoCommand extends Command {
+
+    public static final String COMMAND = "todo";
+    private final String description;
+    private static final String ERROR_MESSAGE = "☹ Sorry, please enter a description for the todo.\n\t" +
+            "Command: todo [description]";
+
+    private ToDoCommand(String description) {
+        this.description = description;
+    }
+
+    @Override
+    public String execute() throws DukeException {
+        Task newTask = new ToDo(description);
+        taskList.addTask(newTask);
+
+        StringBuilder builder = new StringBuilder();
+        builder.append("Got it. I've added this task:\n\t").append(newTask).append("\nNow you have ")
```

Comment:
It might assist readability to break the `.append` into multiple lines since they are new lines, and it makes it clear what is being appended to the `StringBuilder`